### PR TITLE
Clarify pay settle commands

### LINKED ISSUE #346: [PE-D][Tester C] Possible for owed amount to exceed total payable amount
For a 2hr session at 100/hr , this student should owe 200, yet its possible for the student to owe more at 300. Sure if this is the cumulative owed amount (across all sessions), then this is possible, but this information is currently not reflected to the tutor, so this entry is rather confusing as the tutor (user) 

-------------
Labels: `type.FeatureFlaw` `severity.Medium`
original: YangQF2002/ped#6

### LINKED ISSUE #351: [PE-D][Tester A] User able to pay when no amount is owed
Screenshot 2024-11-08 at 17.15.04.png

Unsure if this is an intended feature but users are able to record payment when theres no amount owed. Perhaps a check could be done.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: Rachael-Chan/ped#2

### LINKED ISSUE #379: [PE-D][Tester A] Clarity with UG regarding settle and paid commands
When inputting `settle 7 amount/10`, the paid field and amount owed field increase and decreases accordingly. However, when inputting `pay 7 hr/2`, the paid field increases but owed amount has to be manually entered to change. That being said, readers might be confused with the intended usage of `paid` vs `settle`, since `settle` also increases paid field as well. Perhaps the UG could be more clear with the command overview to state the type of situation where one would use `pay` as opposed to `settle`.

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: Rachael-Chan/ped#3

### PR DESCRIPTION
**Pull request by DennieDan**
Closes #379
Closes #351 
Closes #346 

All the above issues were raised due to the unclear description of `pay` and `settle` commands.

Let's:
* Change the order of command introduction in the UG. In particular, `owe`-`settle`-`pay`
* Briefly explain how each command changes the paid amount and owed amount

To further reduce this ambiguity, a feature will be introduced in the PlanEnhancements in another PR

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -336,6 +308,37 @@ Updates the amount of tuition fee paid and the amount of tuition fee owed by the
 
 </box>
 
+### Receiving payment from a student: `pay`
+
+Updates the amount of tuition fee paid by the specified student after a lesson.
```
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:313]: Maybe can add immediately after a lesson

File: docs/UserGuide.md [hunk 2]
```diff
@@ -318,7 +290,7 @@ Updates the amount of tuition fee owed by a specified student after a lesson. Yo
 
 ### Settling outstanding fees from students: `settle`
 
-Updates the amount of tuition fee paid and the amount of tuition fee owed by the student after the student pays for his/her outstanding tuition fees.
+Settles the amount repaid by the student and adds it to the paid amount. To record payment which is not related to the owed amount, you may use the [`pay` command](#receiving-payment-from-a-student-pay).
```
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:293]: Maybe used owed instead of repaid

**Review by Soumyadip-cmd — COMMENTED** [review]
Some considerations

File: docs/UserGuide.md (hunk 2 above)
Comment (DennieDan) [reply docs/UserGuide.md]: Thanks for the spot. Fixed

**Review by Soumyadip-cmd — COMMENTED** [review]
Lgtm

**Review by Soumyadip-cmd — APPROVED** [review]
Lgtm. Sorry I forgot to approve

**Pull request merged**